build-system/babel-plugins/babel-plugin-transform-simple-array-destructure/test/fixtures/transform/skip-element/output.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+{
+  const _temp = document,
+        second = _temp[1];
+  const _temp2 = document,
+        x = _temp2[0],
+        z = _temp2[2];
+}
```

Comment:
Looks like terser undoes some of this 
```let[,n]=document,[o,,c]=document```

Reply:
Under what options?

Reply:
Pulled the options from our single pass configuration. But cannot reproduce now.

Looking into this again.